scripts/lock.py

```diff
@@ -158,5 +158,10 @@ def parse_args():
         default=None,
         help='OS (distro) version such as "12.10"',
     )
+    parser.add_argument(
+        '--json-query',
+        default=None,
+        help='JSON frag/file of query parms for --list or --brief',
```

Comment:
I had an example in the commit message; I can add it to the code if you want?  Do you mean in the help text?